packages/ppx_expect_nobase/ppx_expect_nobase.v0.17.2.1/opam

```diff
@@ -11,7 +11,7 @@ build: [
 ]
 depends: [
   "ocaml"           {>= "4.14.2" }
```

Comment:
Moving this file will have the effect of removing the previous version from the repo. But,

1. We generally try to avoid removing versions (unless due to our archiving policy)
2. Since the sources are not being changed, it actually seems misleading to change the version.

Was this intentional? If so, could you share the reasoning behind it?